commands/host.vegman

```diff
@@ -65,8 +65,7 @@ function cmd_power {
 # Turn the host on
 function cmd_power_on {
   expect_noarg "$@"
-  ipmitool power on
-  echo "Chassis power is switched on, host is booting up."
+  hostpwrctl on
```

Comment:
Need `exec -a` for `hostpwrctl`'s help to work properly.
The `hostpwrctl` command itself shouldn't be available to the users, so they need to get help in the form of a CLI command.